RealJenkinsRuleInit/pom.xml

```diff
@@ -4,16 +4,15 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.17</version>
+        <version>4.83</version>
         <relativePath />
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>RealJenkinsRuleInit</artifactId>
     <version>0-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
-        <jenkins.version>2.249</jenkins.version>
-        <java.level>8</java.level>
+        <jenkins.version>2.361</jenkins.version>
```

Comment:
version [has no detached plugins as of today,](https://github.com/jenkinsci/jenkins/blob/7aad67832442b8cd1dfaf363a3630d2687ddf3e3/core/src/main/resources/jenkins/split-plugins.txt) and matches the Jenkins version that the j-t-h itself uses

Reply:
this reveals at least one test was relying on this dragging in detached plugins (`instance-identity`) as [it was creating](https://github.com/jenkinsci/jenkins/pull/9351/commits/fe2eaa52bb98d6a4038841aac42f5c28cc70b861) classic inbound agents which would fail with this update until `instance-identity` was added back as a test plugin.

Reply:
I was surprised that the earlier version was not based on an LTS and neither is the new version.  Is there a specific reason this does not use an LTS version?